dev-python/pybind11/files/pybind11-2.13.1-gcc14-fix.patch

```diff
@@ -0,0 +1,22 @@
+From 51c2aa16de5b50fe4be6a0016d6090d4a831899e Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to the upstream commit at the top of the patch.

Reply:
Thanks. Done